torchx/Makefile

```diff
@@ -0,0 +1,26 @@
+
+ifndef LIBTORCH_DIR
+	$(error LIBTORCH_DIR env var is not set. It should point to the directory, where libtorch is installed.)
```

Comment:
Also include the URL to download libtorch too. Something like: "You can download libtorch here: https://pytorch.org/get-started/locally/"